### DISCUSSION TITLE
How to get information from hyperlink and multiple pages.

### DISCUSSION THREAD

**Discussion by dar19** [alirezamika/autoscraper#63] [Q&A]
HI thank you for the wonderful and easy tool for scrapping...
My point is how do we scrape the data from a 
**1) hyperlink from webpage**
url = 'https://www.analyticsvidhya.com/blog/category/machine-learning/'
wanted_list = ['How much Mathematics do you need to know for Machine Learning?']
I want to get the author or any other information from the article and other similar articles. So i will be needed to open the article, get that information and repeat that for all articles.

**2) multi-page website in which we have next (with the multiple pages).**
e.g at the bottom of the main page (url) there is list of webpages like **1 2 3 4...Next** . I need to collect the information from all these pages too.

Thank you in advance.
Regards
Dar

**Comment by Hudson-Reilly** [comment]
Bump
